src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/NoOpFixAllProvider.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    internal sealed class NoOpFixAllProvider : FixAllProvider
```

Comment:
doc why we need this.  why can't htis just be represented with a null fix all provider?

Reply:
Added a doc comment with the explanation.